from mysql.connector.errors import ProgrammingError
from bd import nova_conexao

sql = 'INSERT INTO contatos (nome, tel) VALUES (%s, %s)'
args = (
    ('Ana', '9999-9999'),
    ('Davi', '9999-9999'),
    ('Rhuan', '9999-9999'),
    ('Manuela', '9999-9999')
)

with nova_conexao() as conexao:
    try:
        cursor = conexao.cursor()
        cursor.executemany(sql, args)
        conexao.commit()
    except ProgrammingError as e:
        print(f'Erro: {e.msg}')
    else:
        print(f'Foram inclu√≠dos {cursor.rowcount}  registros!')

